content/docs/config/basics/types.md

```diff
@@ -0,0 +1,58 @@
++++
```

Comment:
I don't like the table design either but couldn't think of a better way to provide a good overview for types and placeholders. Would be happy to change it if you have other ideas but if not, I guess it would be good to open an issue as you said and get back to it later :)